Update Hypershift docs for v0.6 (#2410)
timescale/docs: use-timescale/migration/about-hypershift.md

To see all available configuration options for hypershift, use this command:

```bash
docker run -ti timescale/hypershift:0.6 --help
```

> Hypershift accepts operations it should perform as commands. These commands should be passed as arguments. Supported commands: `clone`

Reviewer: ```suggestion
You can give hypershift commands to perform other operations. 
These commands are passed as arguments. Currently, the only 
supported command is `clone`, which is used to FIXME.
```

I'm not sure if this is right, and it needs a little more exposition.

Author: The `clone` command in Hypershift is basically an instruction to Hypershift to fork/make a full-copy of the contents of a source database into a target database. I think it was misunderstood.

Reviewer: @Harkishen-Singh Updated for you, let me know what you think.

### Configuration flags under `clone`

|Short command|Long command|Example use|Description|
|-|-|-|-|
|-s|--source|-s <SOURCE_URI>|URI to the source database|
|-t|--target|-t <TARGET_URI>|URI to the target database|
||--source-password|--source-password|Password for the source database|
||--target-password|--target-password|Password for the target database|

> ||--parallel|--parallel <NUM_WORKERS>|Number of workers to fetch data parallely from the target database|

Reviewer: ```suggestion
||--parallel|--parallel <NUMBER>|Number of workers that can simultaneously fetch data from the target database|
```

||--verify|--verify|Verify the migrated data by comparing the source and target database|
||--ignore-missing-time-index|--ignore-missing-time-index|Ignore that hypertables may be missing a time index|
||--hide-progress|--hide-progress|Hide migration progress bar|
||--verbose|--verbose|Display verbose logs with progress bar|

> |-c|--config-file|-c <CONFIG_FILE>|Path for advanced hypershift configuration file. For details, please see [hypershift-config]|

Reviewer: ```suggestion
|-c|--config-file|-c <PATH>|Path to an advanced hypershift configuration file. For more information, see [hypershift-config]|
```
